### PR TITLE
#2016235 test modularized

### PR DESCRIPTION
**Pull request by rH4rtinger**

### CODE REVIEW

File: src/test/java/de/adito/aditoweb/nbm/tests/internal/runconfig/TestsRunConfigProviderTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import java.nio.file.Files;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Test for {@link TestsRunConfigProvider}
+ *
+ * @author s.seemann, 01.04.2021
```
Comment (L1nc0ln) [inline src/test/java/de/adito/aditoweb/nbm/tests/internal/runconfig/TestsRunConfigProviderTest.java:23]: I assume author and date are wrong here?

File: src/test/java/de/adito/aditoweb/nbm/tests/internal/runconfig/TestsRunConfigProviderTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    @Override
+    public Observable<Optional<INodeJSEnvironment>> current()
+    {
+      return Observable.just(Optional.of(new INodeJSEnvironment()
+      {
+        @NotNull
+        @Override
+        public File getPath()
+        {
+          return null;
```
Comment (L1nc0ln) [inline src/test/java/de/adito/aditoweb/nbm/tests/internal/runconfig/TestsRunConfigProviderTest.java:90]: Not sure about all these null returns on methods annotated with notnull - I guess they aren't actually called in the tests cases so it does not matter at the moment, but couldn't that lead to some strange errors in the future?

File: src/test/java/de/adito/aditoweb/nbm/tests/internal/runconfig/TestsRunConfigProviderTest.java (hunk 1 above)
Comment (rH4rtinger) [reply src/test/java/de/adito/aditoweb/nbm/tests/internal/runconfig/TestsRunConfigProviderTest.java:23]: the original class is from Stefan, I just renamed the class

File: src/test/java/de/adito/aditoweb/nbm/tests/internal/runconfig/TestsRunConfigProviderTest.java (hunk 2 above)
Comment (rH4rtinger) [reply src/test/java/de/adito/aditoweb/nbm/tests/internal/runconfig/TestsRunConfigProviderTest.java:90]: I changed these null values to an UnsupportedOperationException.

File: src/test/java/de/adito/aditoweb/nbm/tests/internal/runconfig/TestsRunConfigProviderTest.java (hunk 1 above)
Comment (L1nc0ln) [reply src/test/java/de/adito/aditoweb/nbm/tests/internal/runconfig/TestsRunConfigProviderTest.java:23]: ah, I didn't see that, github just showed it as added file and I didn't compare it to the deleted file

**Review by L1nc0ln — APPROVED** [review]

**Pull request merged**